### PR TITLE
SEO-3239 UWP Page with redirect

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,

Please review the changes for UWP Page with redirect.

Regards,
Yvone.

### CODE REVIEW

File: uwp/Scheduler/getting-started.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ This section explains how to create a simple application using SfSchedule contro
 ![UWP Schedule](Getting-Started_images/GettingStarted.png) 
 
 You can download the entire source code of this demo for UWP from    
-here [ScheduleGettingStarted](http://www.syncfusion.com/downloads/support/directtrac/general/ze/GettingStarted_ScheduleUWP1114729068.zip)    
+here [ScheduleGettingStarted](https://help.syncfusion.com/uwp/scheduler/getting-started)
```
Comment (gowthamanthavasiyappan) [inline uwp/Scheduler/getting-started.md:55]: Why you have updated the UG link? Recheck and update.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]

**Comment by gowthamanthavasiyappan** [comment]
Commented

File: uwp/Scheduler/getting-started.md (hunk 1 above)
Comment (Yvone-Atieno) [reply uwp/Scheduler/getting-started.md]: changed

**Review by gowthamanthavasiyappan — APPROVED** [review]

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**